### PR TITLE
Update Register Page

### LINKED ISSUE #614: Add Phone number in register changes and change email to official mail

### PR DESCRIPTION
**Pull request by yashwin**
Changes in the PR
1) Add phone number field in register page
2) Update to official email 
3) Create account link from login page

Fixes #614 
Also need to close https://github.com/appknox/website/pull/223 & ~~https://github.com/appknox/mycroft/pull/908~~ (merged)

### CODE REVIEW

**Comment by cosmosgenius** [comment]
@yashwin this package is not there in npm? https://github.com/Socivo/ember-cli-intl-tel-input.git.
If not can we fork and publish under appknox org?

**Comment by subho007** [comment]
@cosmosgenius yes please. Also resolve the conflicts

**Comment by cosmosgenius** [comment]
@yashwin this is already there in npm https://www.npmjs.com/package/ember-cli-intl-tel-input can you try it?

File: config/environment.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ module.exports = function(environment) {
     ENV.rollbar = {
       enabled: false
     };
-    ENV['host'] = "https://api.appknox.com";
```
Comment (cosmosgenius) [inline config/environment.js]: Maybe even move ENV.socketPath to socket.appknox.io

**Pull request merged**